app/src/main/res/layout/dialog_theme_default.xml

```diff
@@ -2,13 +2,11 @@
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
-
     <RadioGroup
         android:id="@+id/radio_group_themes"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:orientation="vertical">
-
```

Comment:
Should this change be here?
If you are having issues, please open a clean new PR and have only required files

Reply:
Yes. I will open a new PR.